mne_bids/mne_bids.py

```diff
@@ -509,17 +515,55 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         _events_tsv(events, raw, events_tsv_fname, event_id, event_type,
                     verbose)
 
+    # Before writing the neural data, cover some special cases:
     # for FIF, we need to re-save the file to fix the file pointer
     # for files with multiple parts
     if ext in ['.fif', '.fif.gz']:
         raw.save(raw_file_bids, overwrite=overwrite)
+
+    # If NOT fif, check if we got a multifile format
     else:
-        if os.path.exists(raw_file_bids):
-            if overwrite:
-                os.remove(raw_file_bids)
-                sh.copyfile(raw_file, raw_file_bids)
+        # Cover BrainVision multifile format of .vhdr, .eeg, .vmrk
+        if ext in ['.eeg']:
+            # Get future file names
+            fname, ext = _parse_ext(raw_file_bids)
+            assert ext == '.eeg'
+            assert fname + ext == raw_file_bids
+            raw_file_bids_list = [fname + '.eeg',
+                                  fname + '.vhdr',
+                                  fname + '.vmrk']
+            # Get locations of current files
+            fname, ext = _parse_ext(raw_file)
+            assert ext == '.eeg'
+            assert fname + ext == raw_file
+            raw_file_list = [fname + '.eeg',
+                             fname + '.vhdr',
+                             fname + '.vmrk']
+        else:
+            # All other data are single file formats
+            raw_file_bids_list = [raw_file_bids]
+            raw_file_list = [raw_file]
+
+        # Now copy&move or overwrite the files for the new BIDS directory
+        for raw_file, raw_file_bids in zip(raw_file_list, raw_file_bids_list):
+            if os.path.exists(raw_file_bids):
+                if overwrite:
+                    os.remove(raw_file_bids)
+                    sh.copyfile(raw_file, raw_file_bids)
+                else:
+                    raise ValueError('"%s" already exists. Please set'
+                                     ' overwrite to True.' % raw_file_bids)
             else:
-                raise ValueError('"%s" already exists. Please set overwrite to'
-                                 ' True.' % raw_file_bids)
+                if verbose:
+                    print('Writing data files to %s' % raw_file_bids)
+                # First assume we have single files
+                try:
+                    sh.copyfile(raw_file, raw_file_bids)
```

Comment:
@jasmainak for this most recent commit I need a bit of MEG experise.

What I changed is that if the actual data (not some BIDS meta data) is NOT present in the to-be-done BIDS-directory, it is getting copied there. This already needed a few tweaks to cover e.g., the multifile system for BrainVision (see code), which seems to work. Another tweak concerned CTF data, which apparently is always saved in a directory ending on `.ds`
____________
Now the issue is with KIT and BTi data ... see the travis [output](https://travis-ci.org/mne-tools/mne-bids/builds/405795025?utm_source=github_status&utm_medium=notification)

Due to the added line 561, I am now writing data to the BIDS dir, which has extensions that are not part of BIDS.

Before adding that line, it seems to me that no data was moved to the freshly made BIDS directory. Is that correct? How can we deal with this now?

Reply:
okay your travis issue seems related to this issue on bids validator: https://github.com/INCF/bids-validator/pull/485 ? Am I right?

Reply:
Mmmh yes, it seems like it although I lack the comprehensive knowledge about the file formats to be 100% sure ...

Reply:
maybe you can test it on your files and then put some subtle pressure on @chrisfilo to merge it :)

Reply:
See if the PR actually fixes your problem. It could be a good check also for that PR

Reply:
I copied your changes from the PR into my local source and then ran `sudo npm install -g .` ... the error persists

Reply:
no time to look right now :( I need to run now. I'll look later if you don't figure out what's the issue

Reply:
merged and released

Reply:
Thanks !